src/ResourceManager/Compute/Commands.Compute/Generated/AvailabilitySet/AvailabilitySetGetMethod.cs

```diff
@@ -83,20 +82,13 @@ protected void ExecuteAvailabilitySetGetMethod(object[] invokeMethodInputParamet
             if (!string.IsNullOrEmpty(resourceGroupName) && !string.IsNullOrEmpty(availabilitySetName))
             {
                 var result = AvailabilitySetsClient.Get(resourceGroupName, availabilitySetName);
-                var psObject = new PSAvailabilitySet();
-                Mapper.Map<AvailabilitySet, PSAvailabilitySet>(result, psObject);
-                WriteObject(psObject);
+                WriteObject(result);
             }
             else if (!string.IsNullOrEmpty(resourceGroupName))
             {
                 var result = AvailabilitySetsClient.List(resourceGroupName);
                 var resultList = result.ToList();
-                var psObject = new List<PSAvailabilitySetList>();
-                foreach (var r in resultList)
-                {
-                    psObject.Add(Mapper.Map<AvailabilitySet, PSAvailabilitySetList>(r));
-                }
-                WriteObject(psObject, true);
+                WriteObject(resultList, true);
```

Comment:
This code does not produce Powershell cmdlets.  The method is for the generalized Powershell cmdlets (Invoke cmdlets), but we don't generate this cmdlets anymore.  So this change does not have any effect.